libass/ass_cache.c

```diff
@@ -446,21 +545,35 @@ void ass_cache_dec_ref(void *value)
         return;
 
     Cache *cache = item->cache;
-    if (cache) {
```

Comment:
Original idea was to use `ref_count` as a memory barrier for `cache` so this line would be safe.